generators/app/index.js

```diff
@@ -188,7 +194,8 @@ module.exports = generators.Base.extend({
         this.prompt({
           name: 'githubAccount',
           message: 'GitHub username or organization',
-          default: username
+          default: username,
+          when: !this.options.githubAccount
```

Comment:
We should actually skip the whole `githubUsername` block is the `githubAccount` option is provided. No need to make a server request if we'll skip it.